x-pack/plugins/observability/public/pages/alert_details/components/page_title.tsx

```diff
@@ -40,6 +37,7 @@ import {
 
 export interface PageTitleProps {
   alert: TopAlert | null;
+  alertStatus: string | undefined;
```

Comment:
As mentioned above, I think having this as `alertStatus: AlertStatus` and case the string in the parent component makes the code more readable. Also, I am not sure about the `undefined` scenario and what should be shown in that case 🤔

Reply:
refactored in https://github.com/elastic/kibana/pull/174153/commits/416b995c83d9e1113343e92649ed6a1d78bdc391